Cargo.toml

```
@@ -34,6 +34,7 @@ itertools = "0.8.2"
 lazy_static = "1.4.0"
 log = "0.4.8"
 prettytable-rs = { version = "0.8.0", default-features = false }
+sentry = { version = "0.18.1", features = ["with_env_logger"] }
```

Can we make this dependency optional so that people can still compile the node without any telemetry? Although I don't know what's the command to disable some feature when compiling.

```suggestion
sentry = { version = "0.18.1", features = ["with_env_logger"], optional = true }
```